src/sider/oppfolgingsplan/oppfolgingsplaner/OppfolgingsplanerOversikt.tsx

```diff
@@ -89,9 +115,31 @@ export default function OppfolgingsplanerOversikt(props: Props) {
   const hasNoInactivePlans =
     inaktivePlaner.length === 0 && oppfolgingsplanerLPSProcessed.length === 0;
 
+  const activeNarmesteLedere = !!latestOppfolgingstilfelle
+    ? activeNarmesteLederForCurrentOppfolgingstilfelle(
+        currentLedere,
+        latestOppfolgingstilfelle
+      )
+    : [];
```

Comment:
Burde denne sjekken vært `!!latestOppfolgingstilfelle && hasActiveOppfolgingstilfelle` også?

Reply:
Sjekker det jo etterpå, men heller ikke vits å finne frem aktive nærmeste ledere om vi ikke gjør det for et gjeldende oppfølgingstilfelle 😅 Jeg endrer 😊